README.md

```diff
@@ -1,29 +1,145 @@
+*NOTE: Checked box means that playbook is working and supported, unchecked box means that playbook needs stabilization.*
 
+**Tested on CentOS Linux release 7.4 (Core), OpenShift 3.7 and Ansible 2.4.2**
 
-## Install KubeVirt on existing OpenShift cluster
+## List of playbooks
+
+| Playbook name | Description |
+| ------------- | ----------- |
+| install-kubevirt-on-openshift.yml | This playbook installs KubeVirt on existing OpenShift cluster. |
+| deploy-kubernetes.yml | This playbook deploys Kubernetes cluster on given machines. |
+| deploy-openshift.yml | This playbook deploys OpenShift cluster on given machines. |
+| control.yml | This is top level playbook which deploys Kubernetes or OpenShift cluster using Lago provider and installs KubeVirt on top of it. |
+
+## Requirements
+
+Install depending roles, and export `ANSIBLE_ROLES_PATH`
+
+```bash
+$ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
+$ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
+```
+
+For OpenShift deployment clone [**OpenShift Ansible project**][openshift-ansible-project]
+
+```bash
+$ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
+```
+
+## (Optional) Setup a cluster
+
+This section describes how to setup cluster on given machines. There are two
+playbook which you can use depends on what cluster you want to bring up.
+
+
+### General resources for cluster
+
+There are three key cluster components which needs to be deployed.
+
+* **master**
+
+  The master is the host or hosts that contain the master components,
+  including the API server, controller manager server, and etcd.
+  The master manages nodes in its Kubernetes cluster and schedules pods
+  to run on nodes.
+
+* **etcd**
+
+  **etcd** stores the persistent master state while other components watch
+  **etcd** for changes to bring themselves into the desired state.
+
+* **node**
+
+  A node provides the runtime environments for containers.
+  Each node in a Kubernetes cluster has the required services
+  to be managed by the master. Nodes also have the required services
+  to run pods, including the Docker service, a kubelet, and a service proxy.
+
+There are two basic environment scenarios on how a cluster can be deployed.
+If you need more information please read
+[documentation](https://docs.openshift.org/latest/install_config/install/planning.html).
+
+* All-in-one
+  * all components on single machine
+* Single master and multiple nodes
+  * master & etcd is deployed on single machine
+  * nodes are separated machines
+
+For minimal hardware requirements please follow
+[documentation](https://docs.openshift.org/latest/install_config/install/prerequisites.html) .
+
+If you take a look at [inventory](./inventory) file, you can find three
+host groups which you need fill in with your machines according to prefered
+topology.
+
+For example if you choose `all-in-one` topology you can read the following inventory file.
```

Comment:
Ansible service broker (which is required for installing APBs) need additional storage to be configured https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-openshift-ansible-broker

Either add storage configurations or specify that the broker will not be available.

Reply:
@gbenhaim ok I will add `nfs` group and update inventory